en/build-multi-aws-eks.md

```diff
@@ -0,0 +1,426 @@
+---
+title: Build Multiple Interconnected AWS EKS Clusters
+summary: Learn how to build multiple interconnected AWS EKS clusters and prepare for deploying a TiDB cluster across multiple EKS clusters.
+---
+
+# Build Multiple Interconnected AWS EKS Clusters
+
+This document describes how to create multiple AWS EKS clusters and configure network peering between these clusters. These interconnected clusters can be used for [deploying TiDB clusters across multiple Kubernetes clusters](deploy-tidb-cluster-across-multiple-kubernetes.md). The example in this document shows how to configure three-cluster network peering.
+
+If you need to deploy TiDB on a single AWS EKS cluster, refer to [Deploy TiDB on AWS EKS](deploy-on-aws-eks.md).
+
+## Prerequisites
+
+Before you deploy EKS clusters, make sure you have completed the following preparations:
+
+- Install [Helm 3](https://helm.sh/docs/intro/install/). You need to use Helm to install TiDB Operator.
+
+- Complete all steps in [Getting started with Amazon EKS – eksctl](https://docs.aws.amazon.com/eks/latest/userguide/getting-started-eksctl.html).
+
+    This tutorial includes the following tasks:
+
+    - Install and configure the AWS CLI (`awscli`).
+    - Install and configure the CLI for creating Kubernetes clusters (`eksctl`).
+    - Install the Kubernetes CLI (`kubectl`)
+
+- Grant AWS Access Key the [minimum permissions required for `eksctl`](https://eksctl.io/usage/minimum-iam-policies/) and the [permissions required for creating a Linux bastion](https://aws-quickstart.github.io/quickstart-linux-bastion/#_aws_account).
+
+To verify whether you have correctly configured the AWS CLI, run the `aws configure list` command. If the output shows the values of `access_key` and `secret_key`, you have successfully configured the AWS CLI. Otherwise, you need to reconfigure the AWS CLI.
+
+## Step 1. Start the Kubernetes cluster
+
+Define the configuration files of three EKS clusters as `cluster_1.yaml`, `cluster_2.yaml`, and `cluster_3.yaml`, and create three clusters using `eksctl`.
+
+1. Define the configuration file of Cluster 1, and create Cluster 1.
```

Comment:
```suggestion
1. Define the configuration file of cluster 1, and create cluster 1.
```

Reply:
This is a global comment. It seems that we can use cluster 1, cluster 2, cluster 3 directly.

Reply:
Thanks. Done!